### ISSUE TITLE
[responsiveness] Mobile view of developer website is not laid out appropriately

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#201]
URL: https://developers.openreferraluk.org/

Most of the issues are related to flex applied to small screens.

<img width="317" alt="Screenshot 2021-07-05 at 16 49 04" src="https://user-images.githubusercontent.com/2459151/124496118-ec195d80-ddb0-11eb-830b-34811c3c51b3.png">

- The logo should have a max-width of 40% less than 768px.
- The class `row` should not have `display: flex` less than 768px.

**Comment by UsmanAfzal** [comment]
Checked and approved.
